pom.xml

```diff
@@ -137,6 +142,11 @@
       <version>1.19.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <version>1.7.13</version>
+    </dependency>
```

Comment:
Is this required for the plugin to work?

Reply:
The `org.marc4j.callnum` package does not explicitly reference `org.slf4j`, but I can't say that means the plugin would. Nor is the `slf4f-api.jar` included with the MARC4J distribution.

From a license perspective, `org.slf4j` license terms are identical to the MIT License, so there's no problem there.

Reply:
@todolson Apologies for not being more specific. By plugin, I was referring to the maven plugin used to generate the license information for inclusion in the module, not the MARC4J library

Reply:
It looks like some of transitive dependencies or libraries uses this logging facet:
![image](https://user-images.githubusercontent.com/63074477/109309190-a02b3a80-784b-11eb-9829-2b1939bcdb9a.png)
Taking into account that we leverage log4j provider in code in general, I thought it will be reasonable to specify bridge implementation  for it.

Reply:
Adding slf4j-log4j12 is unrelated to adding the license-maven-plugin. In future it is better to create a separate pull request for unrelated changes to allow for independent code reviews.